wies/projects/templates/colleague_list.html

```diff
@@ -2,49 +2,184 @@
 
 {% block content %}
 
-<h1 class="utrecht-heading-1">Collega's</h1>
-
-<form id="assignment-filter-form"
-      hx-get="/colleagues/"
-      hx-target="#colleague_table"
-      hx-trigger="change, keyup delay:300ms from:#name-search"
-      hx-swap="outerHTML"
-      class="rvo-layout-grid rvo-layout-gap--md rvo-layout-grid-columns--three"
-      style="margin-bottom: 2em;">
-    <div class="rvo-layout-column rvo-layout-gap--sm">
-        <label for="name-search" class="rvo-label">Naam</label>
-        <input type="text" 
-               id="name-search" 
-               name="name" 
-               value="{{ request.GET.name|default:'' }}"
-               placeholder="Zoek op naam..."
-               class="utrecht-textbox utrecht-textbox--html-input">
-    </div>
-    <div class="rvo-layout-column rvo-layout-gap--sm">
-        <label for="skill-filter" class="rvo-label">Rollen:</label>
-        <select id="skill-filter" name="skill"
-                class="js-skills-select-single utrecht-select utrecht-select--html-select">
-        </select>
-    </div>
-    <div class="rvo-layout-column rvo-layout-gap--sm" style="align-self: end; text-align: right;">
-        <a href="/colleagues/new">
-          <button
-            class="utrecht-button utrecht-button--primary-action utrecht-button--rvo-md utrecht-button--icon-before"
-            type="button"
-          >
-            <span
-              class="utrecht-icon rvo-icon rvo-icon-plus rvo-icon--md rvo-icon--wit"
-              role="img"
-              aria-label="Plus"
-            ></span>
-            Nieuwe collega toevoegen
-          </button>
-        </a>
-    </div>
-</form>
+<div style="display: flex; justify-content: space-between; align-items: center; margin-bottom: 1rem; margin-top: 1rem;">
+  <h1 style="margin: 0;">Collega's</h1>
+</div>
+
+<!-- Filters Section -->
```

Comment:
many of the filters don't work:


Uploading Screen Recording 2025-08-04 at 10.32.37.mov…